packages/fabrix/src/renderers/fields.tsx

```diff
@@ -1,24 +1,23 @@
 import { createElement, useCallback, useMemo } from "react";
-import { ViewFieldSchema } from "@directive/schema";
-import { FieldConfigWithMeta } from "@readers/shared";
 import { FabrixContextType } from "@context";
 import { useDataFetch, Value } from "../fetcher";
 import { RendererCommonProps } from "../renderer";
 import {
   assertObjectValue,
   buildClassName,
   CommonFabrixComponentRendererProps,
+  FieldConfigByType,
   getFieldConfigByKey,
   Loader,
   resolveFieldTypesFromTypename,
 } from "./shared";
 
-type ViewField = FieldConfigWithMeta<ViewFieldSchema>;
-type Fields = Array<ViewField>;
```

Comment:
here was a bit hard to know where these types come from, so use type inference instead.